src/pipeline_integration.jl

```diff
@@ -0,0 +1,240 @@
+import Plots
+using Plots.RecipePipeline
+
+# Define overrides for RecipesPipeline hooks.
+
+RecipePipeline.RecipesBase.apply_recipe(plotattributes::Plots.AKW, ::Type{T}, ::AbstractPlotting.Scene) where T = throw(MethodError("Unmatched plot type: $T")) # TODO: loosen this restriction and move to RecipesBase
```

Comment:
I think that this should just be
```suggestion
RecipesBase.apply_recipe(plotattributes::Plots.AKW, ::Type{T}, ::AbstractPlotting.Scene) where T = throw(MethodError("Unmatched plot type: $T"))
```

Reply:
Thanks, that can go - it needs to be matched to the Scene as well.